conf/uno.conf

```
@@ -102,8 +102,8 @@ if [[ "${HADOOP_VERSION}" = 1.* ]]; then
    exit 1
 fi
 
-if [[ -z "$ACCUMULO_REPO" && "${ACCUMULO_VERSION}" = 1.* && ! "${HADOOP_VERSION}" = 2.* ]]; then
-   echo "ERROR from uno.conf : When using Accumulo 1.x, expect Hadoop 2.x not $HADOOP_VERSION" 1>&2
+if [[ -z "$ACCUMULO_REPO" && "${ACCUMULO_VERSION}" =~ ^1\.[[:digit:]]\.[[:digit:]]+$ && ! "${HADOOP_VERSION}" = 2.* ]]; then
```

Can remove unnecessary quotes/braces, and `[0-9]` is a shorter way of matching digits, and `[.]` matches a dot without a backslash (because backslashes are annoying, especially when they get nested inside strings and you have multiple escape character layers).

```suggestion
if [[ -z $ACCUMULO_REPO && $ACCUMULO_VERSION =~ ^1[.][0-9][.][0-9]+$ && ! $HADOOP_VERSION = 2.* ]]; then
```

---

I updated to add the changes if you'd like to merge. Thanks.